### PR TITLE
feat: add different rankings in the All Time tab

### PR DESCRIPTION
**Pull request by ManuViola77**
## Links:

[tab All Time para el resto de los rankings](https://github.com/users/ManuViola77/projects/1/views/1#:~:text=tab%20All%20Time%20para%20el%20resto%20de%20los%20rankings)

## What & Why:

This PR adds different types of ranking in the All Time tab. It makes it easy to add a new ranking option, just need to add an object in the `RANKING_VALUES` type with this format: 

```
 {
    value: 'value',
    label: 'label,
    getNumericValue: ({ interestedFieldsFromStatistics }) => calculatedValueWithInterestedFields,
    getRightText: ({ interestedFieldsFromStatistics }) => calculatedTextWithInterestedFields,
    getSuffix: ({ interestedFieldsFromStatistics }) => calculatedTextWithInterestedFields,
    sort: (firstValue, secondValue) => calculatedSortFunction,
  }
```

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/components/common/Select/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+const activeColor = '#1f1f1f';
+const backgroundColor = '#538d4e';
+const color = 'white';
+const focusedColor = '#1f1f1f';
+const hoverColor = '#c6d4c5';
+const selectedColor = '#3d6b39';
+
+const Select = ({ ...props }) => {
+  const colorStyles = {
```
Comment (jpirazusta) [inline src/components/common/Select/index.js:12]: We can move this object out of the component. I can do that when I commit the thing about the blue border

File: src/hooks/useRankingData.js [hunk 2]
```diff
@@ -113,14 +118,50 @@ const useRankingData = () => {
       state: { email, name, photo },
     });
 
+  const onChangeSelectedRanking = newSelectedRanking => {
+    if (newSelectedRanking !== selectedRanking) {
+      setSelectedRanking(newSelectedRanking);
+      var newRankingData = [...rankingData];
```
Comment (jpirazusta) [inline src/hooks/useRankingData.js]: Why `var`?

Comment (ManuViola77) [reply src/hooks/useRankingData.js:124]: because Im assigning it later

image

Comment (ManuViola77) [reply src/hooks/useRankingData.js]: because I want to see the replay and check if it was offside or not

Comment (jpirazusta) [reply src/hooks/useRankingData.js]: 😂 😂 👏

**Review by jpirazusta — APPROVED** [review]

**Pull request merged**